Document/Poll.php

```diff
@@ -28,13 +29,17 @@ class Poll extends Content
     protected $title;
 
     /**
-     * @var string
+     * @var array
```

Comment:
Can you change this type-hint to `Option[]`? For this is the only object that can be stored in this attribute.
Can you do the same for the methods: `setOptions()` and `getOptions()`.